content/about-us/contents.lr

```diff
@@ -0,0 +1,111 @@
+_model: generic_page
+---
+title: About Us
+---
+content:
+
+#### text ####
+section_header: Who we are
+----
+section_level: 1
+----
+markdown:
+
+We are the inventors of [PyMC3][pymc3],
+the leading toolbox to solve advanced data science problems.
+We have launched a consultancy to turn our expertise into your advantage.
+Our decades of experience in Bayesian modeling
+allows us to come up with unique and impactful solutions
+to your most challenging business problems.
+
+[pymc3]: https://docs.pymc.io
+
+Our team consists of PhDs, statisticians,
+mathematicians, neuroscientists, social scientists
+and a former Space-X rocket scientist.
+
+We work according to the following principles:
+
+1. Provide value: We focus primarily on generating outcomes that positively affect your bottom line.
+1. Iterative development: We always start small and go for the low-hanging fruits first. From this bridgehead we venture into developing more complex solutions to bigger problems.
+1. Transparency & frequent communication: Rather than develop a finished product in isolation we keep close touch with our clients. All of our customers are invited to our company chat for direct communication as well as the GitHub repo where we develop and project-manage so you can exactly see what the current status is.
+
+#### text ####
+section_header: What do we need to solve your problem?
+----
+section_level: 1
+----
+markdown:
+
+We usually start with an introductory call to introduce ourselves
+and get an understanding of the challenges you and your company face.
+From there we identify the simplest but most impactful project
+which we then want to understand in more detail.
+
+We usually require a data set,
+though we have also developed models with mock data
+that the client can then fit themselves on their data.
+
+#### text ####
+section_header: How Bayesian statistics can help you
+----
+section_level: 1
+----
+markdown:
+
+Bayesian statistics, as implemented in [PyMC3](https://docs.pymc.io),
+allows you to solve data science problems of varying complexity
+that you can hardly solve any other way.
+
+The tool is highly versatile and is being used successfully by various companies.
+For example, SpaceX used PyMC3 to optimize its supply chains
+(as explained in this [blog post](https://twiecki.io/blog/2019/01/14/supply_chain/)).
+But it's also being used to [find planets outside of our solar-system](https://github.com/exoplanet-dev/exoplanet),
+[detect earthquakes](https://github.com/hvasbath/beat),
+[predict elections](https://github.com/AlexAndorra/pollsposition_models)
+or [estimate the spread of COVID-19 on rt.live](https://rt.live/).
+The PyMC3 journal article is the most cited paper on PeerJ where it was published.
```

Comment:
There should be a link here. The google doc has https://peerj.com/articles/cs-55/?utm_source=mailshot&utm_medium=email&utm_campaign=cs_top_5_views, but given that the emphasis is not so much on having a paper on it but on the paper being the most cited one, I'd suggest using https://peerj.com/blog/post/115284882768/peerj-computer-sciences-most-cited-paper-probabilistic-programming-in-python-using-pymc3-author-interview-with-thomas-wiecki/

Reply:
Wonderful, thanks @OriolAbril!